src/main/java/org/veupathdb/service/eda/ds/plugin/AbstractPlugin.java

```diff
@@ -239,8 +239,12 @@ protected String getVariableDataShape(List<VariableSpec> vars, int index) {
   protected String singleQuote(String unquotedString) {
     return "'" + unquotedString + "'";
   }
+
+  protected String getVoidEvalFreadCommand(String fileName, VariableSpec... vars) {  
+    return getVoidEvalFreadCommand(fileName, new ListBuilder().addAll(vars).toList());
+  }  
   
-  protected String getVoidEvalFreadCommand(String fileName, VariableSpec... vars) {    
+  protected String getVoidEvalFreadCommand(String fileName, List<VariableSpec> vars) {
```

Comment:
Probably don't need both here.  If you keep the varargs one and have a list in the calling code, use List.toArray()